checklist.py

```diff
@@ -144,6 +146,67 @@ def _system_call(self, cmd):
         return stdout, stderr, return_value
 
 
+class CopyrightHeadersValidator(RepoValidator):
+    """Flag library files with non-standard copyright headers
+
+    See the current standard for scikit-bio's copyright headers at
+    ``http://scikit-bio.org/docs/latest/development/new_module.html``
+
+    Parameters
+    ----------
+    skip_dirs : iterable of str, optional
+        Directory names to skip during validation. Defaults to skipping any
+        directories named ``'data'`` or ``'__pycache__'`` (and anything
+        contained within them).
+
+    """
+
+    reason = "Files non-conforming to standard copyright headers"
+
+    COPYRIGHT_HEADER = """\
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+"""
+
+    def __init__(self, skip_dirs=None):
+        if skip_dirs is None:
+            skip_dirs = {'data', '__pycache__'}
+        self.skip_dirs = set(skip_dirs)
+
+    def _validate(self, root, dirs, files):
+        for skip_dir in self.skip_dirs:
+            if skip_dir in dirs:
+                dirs.remove(skip_dir)
+
+        invalid_files = []
+        for _file in files:
+            if not _file.endswith('.py'):
+                continue
+            pos = 0
+            f = open(os.path.join(root, _file))
+            tokens = list(tokenize.generate_tokens(f.readline))
+            if self.is_shebang(tokens[pos][1]):
+                pos += 3
+            if tokens[pos][0] == 3:  # module docstring
+                pos += 3
+            cheader = ''.join(
+                    map(lambda x: x[1], tokens[pos:pos + 14]))
+            if cheader != self.COPYRIGHT_HEADER or tokens[pos + 14][0] != 54:
+                invalid_files.append(f.name)
+            f.close()
+
+        return invalid_files
+
+    @staticmethod
+    def is_shebang(line):
+        return line == '#!/usr/bin/env python'
```

Comment:
Shebang lines can have spaces, ie
```
#! /usr/bin/env python
```
is a valid one that would fail the test. In fact, the space is used by the [python docs](https://docs.python.org/3/using/windows.html#shebang-lines).

A more robust approach would be to check that it's the first line of the script and that it `.startswith('#!')`.